zebrad/src/components/mempool/storage/verified_set.rs

```diff
@@ -54,11 +54,22 @@ impl Drop for VerifiedSet {
 }
 
 impl VerifiedSet {
-    /// Returns an iterator over the transactions in the set.
+    /// Returns an iterator over the [`UnminedTx`] in the set.
+    //
+    // TODO: make the transactions() method return VerifiedUnminedTx,
+    //       and remove the full_transactions() method
     pub fn transactions(&self) -> impl Iterator<Item = &UnminedTx> + '_ {
         self.transactions.iter().map(|tx| &tx.transaction)
     }
 
+    /// Returns an iterator over the [`VerifiedUnminedTx`] in the set.
+    ///
+    /// Each [`VerifiedUnminedTx`] contains an [`UnminedTx`],
+    /// and adds extra fields from the transaction verifier result.
+    pub fn full_transactions(&self) -> impl Iterator<Item = &VerifiedUnminedTx> + '_ {
```

Comment:
```suggestion
    #[cfg(feature = "getblocktemplate-rpcs")]
    pub fn full_transactions(&self) -> impl Iterator<Item = &VerifiedUnminedTx> + '_ {
```

Reply:
We'll need this for tickets #5335 and #5336, which are independent of the  `getblocktemplate-rpcs` feature.